app/src/main/java/com/gleam/kiwi/viewModel/LoginViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package com.gleam.kiwi.viewModel
+
+import androidx.lifecycle.ViewModel
+import com.gleam.kiwi.model.User
+import com.gleam.kiwi.net.KiwiClient
+import com.gleam.kiwi.net.KiwiService
+import com.gleam.kiwi.net.KiwiServiceInterFace
+import kotlinx.coroutines.runBlocking
+
+class LoginViewModel : ViewModel() {
+    private val service = KiwiService().create(KiwiServiceInterFace::class.java)
+    private val client = KiwiClient(service)
+
+    fun signIn(username: String, password: String) {
+        val user = User(username, password)
+        runBlocking {
```

Comment:
`runBlocking` should be use to debug and test. Look at `viewModelScope` in `androidx.lifecycle:lifecycle-viewmodel-ktx`.